### ISSUE TITLE
after installing ubuntu, keyboard does not work

### ISSUE THREAD

**Issue by uygwfd254** [reynhout/chrx#59]
After I install the latest ubuntu, 18.04 lts. and boot it up, my keyboard doesn't work. I can log in with on screen keyboard, but the on screen keyboard is very limited without ctrl, shift, and arrow keys.

how do i fix this?

**Comment by reynhout** [comment]
What model Chromebook? What firmware?

**Comment by uygwfd254** [comment]
i have asus c300sa. my chromebook's version is Version 66.0.3359.203 (Official Build) (64-bit).

**Comment by reynhout** [comment]
So, TERRA. And RW_LEGACY from MrChromebox, I assume?

Keyboard might not work on Braswell in Ubuntu.

The issue is fixed in GalliumOS 2.2 (nightly) and 3.0 (alpha).

To install the GalliumOS 2.2 nightly:
```
cd ; curl -O https://chrx.org/go && sh go -r nightly .....
```

To install GalliumOS 3.0 alpha (very stable on Braswell):
```
export CHRX_WEB_ROOT=https://chrx.org/dev
cd ; curl -O $CHRX_WEB_ROOT/go && sh go -r alpha ......
```

**Comment by uygwfd254** [comment]
ok, thx. i'll try
